<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>聊天室</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #363636;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="leftSidebar" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #2B2B2B;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="channelHeader">
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;
font-size: 18px;
padding: 10px;</string>
         </property>
         <property name="text">
          <string>大哲聊</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="channelList">
         <property name="styleSheet">
          <string notr="true">QListWidget {
    background-color: #2B2B2B;
    border: none;
}
QListWidget::item {
    padding: 5px;
    color: #FFFFFF;
}
QListWidget::item:selected {
    background-color: #404040;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="chatArea" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #363636;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="chatHeader">
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;
font-size: 16px;
padding: 10px;</string>
         </property>
         <property name="text">
          <string>居然还有威望</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="messageArea">
         <property name="styleSheet">
          <string notr="true">QListWidget {
    background-color: #363636;
    border: none;
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="inputArea" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="messageInput">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
    background-color: #2B2B2B;
    border: 1px solid #404040;
    border-radius: 4px;
    color: #FFFFFF;
    padding: 8px;
}</string>
            </property>
            <property name="placeholderText">
             <string>聊聊点什么吧~</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendButton">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: #2B2B2B;
    border: 1px solid #404040;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #404040;
}
QPushButton:pressed {
    background-color: #505050;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="rightSidebar" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #2B2B2B;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="memberHeader">
         <property name="styleSheet">
          <string notr="true">color: #FFFFFF;
font-size: 14px;
padding: 10px;</string>
         </property>
         <property name="text">
          <string>群成员(7/53)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="memberList">
         <property name="styleSheet">
          <string notr="true">QListWidget {
    background-color: #2B2B2B;
    border: none;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>